src/main/scala/faces/settings/FacesSettings.scala

```diff
@@ -27,12 +27,17 @@ abstract class FacesSettings{
   val imageDimensions: ImageDimensions
   val defaultParameters: DefaultParameters
   val landmarkTags: IndexedSeq[String]
+  val occlusion: Occlusion
 }
 
 
 case class OutputLocation(
                            outPath: String
                          ) {
+  def inTexturesPath: String = outPath + "../textures/"
```

Comment:
please put input path not in Outputlocations. use same ideas for path of input as for bip, model or backgrounds